src/api/storage/v0/storage.test.ts

```diff
@@ -341,4 +341,75 @@ describe('Storage Client Tests', () => {
       });
     });
   });
+
+  describe('Given nitric.api.storage.StorageClient.ListFiles throws an error', () => {
+    const MOCK_ERROR = {
+      code: 2,
+      message: 'UNIMPLEMENTED',
+    };
+
+    let listFilesMock;
+
+    beforeAll(() => {
+      listFilesMock = jest
+        .spyOn(GrpcStorageClient.prototype, 'listFiles')
+        .mockImplementation((_, callback: any) => {
+          callback(MOCK_ERROR, null);
+
+          return null as any;
+        });
+    });
+
+    afterAll(() => {
+      jest.resetAllMocks();
+    });
+
+    test('Then StorageClient.delete should reject', async () => {
```

Comment:
should this be StorageClient.ListFiles should reject?

Reply:
fixed